plugin/evm/atomic/test_tx.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/ava-labs/coreth/params/extras"
 )
 
+const testCodecVersion = 0
+
 var TestTxCodec codec.Manager
```

Comment:
Not related to this change, but I have come to the conclusion during testing in the P-chain and X-chain that having test codecs is a bad idea. IMO we should strive to use real transactions and the real codec.

Reply:
I agree, but I'd imagine that requires a whole another refactor to all atomic tests. I added a TODO, we can try to circle back to address this (and bunch of other test codecs)